IFC 4.3/Align_2/UT_Align_2.ifc

```diff
@@ -0,0 +1,72 @@
+#6=IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#7=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#8=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#10=IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.017453293),#8);
+#11=IFCCONVERSIONBASEDUNIT(#6,.PLANEANGLEUNIT.,'Degrees',#10);
+#12=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#13=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#14=IFCUNITASSIGNMENT((#7,#11,#12,#13));
+#15=IFCPROJECT('1uDztrCefFdhoVWQqcuKEd',$,'Dibujo2',$,$,$,$,(#20),#14);
+#16=IFCDIRECTION((1.,0.,0.));
+#17=IFCDIRECTION((0.,0.,1.));
+#18=IFCCARTESIANPOINT((0.,0.,0.));
+#19=IFCAXIS2PLACEMENT3D(#18,#17,#16);
+#20=IFCGEOMETRICREPRESENTATIONCONTEXT('Body','Model',3,$,#19,$);
+#21=IFCDIRECTION((1.,0.,0.));
+#22=IFCDIRECTION((0.,0.,1.));
+#23=IFCCARTESIANPOINT((0.,0.,0.));
+#24=IFCAXIS2PLACEMENT3D(#23,#22,#21);
+#25=IFCLOCALPLACEMENT($,#24);
+#26=IFCSITE('3ntLMNjeHB3xGi3Mk2pVut',#5,'Align_2',$,$,#25,$,$,.ELEMENT.,$,$,$,$,$);
+#27=IFCRELAGGREGATES('2ob8IRSrj7VwmBM2VTtxyH',#5,$,$,#15,(#26));
+#28=IFCALIGNMENT('0xvQy79tbBEwdf5zr3k8vJ',$,'Alignment2',$,$,$,$,#29,$);
+#29=IFCALIGNMENTCURVE(#30,$,$);
+#30=IFCALIGNMENT2DHORIZONTAL(0.,(#31,#34,#37,#40,#43,#46,#49,#52,#55));
+#31=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#32);
+#32=IFCLINESEGMENT2D(#33,0.4936559192503,688.33801902);
+#33=IFCCARTESIANPOINT((488761.49743402,4217495.77914699));
+#34=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#35);
+#35=IFCTRANSITIONCURVESEGMENT2D(#36,0.4936559192503,834.76720476,0.,0.,.F.,.F.,.CLOTHOIDCURVE.);
+#36=IFCCARTESIANPOINT((489367.65229605,4217821.94706641));
+#37=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#38);
+#38=IFCTRANSITIONCURVESEGMENT2D(#39,0.115483092892613,1099.36986773,0.,0.,.F.,.F.,.CLOTHOIDCURVE.);
+#39=IFCCARTESIANPOINT((490141.66542084,4218120.157764));
+#40=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#41);
+#41=IFCTRANSITIONCURVESEGMENT2D(#42,5.90062318802994,928.81668938,0.,972.83675225,.F.,.T.,.CLOTHOIDCURVE.);
+#42=IFCCARTESIANPOINT((491203.48741693,4217886.17009175));
+#43=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#44);
+#44=IFCCIRCULARARCSEGMENT2D(#45,0.0948132917446204,393.90380198,972.83675225,.T.);
+#45=IFCCARTESIANPOINT((492100.01196238,4217682.1608079));
+#46=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#47);
+#47=IFCTRANSITIONCURVESEGMENT2D(#48,0.499715554322829,646.6491343,972.83675225,1387.18510508,.T.,.T.,.CLOTHOIDCURVE.);
+#48=IFCCARTESIANPOINT((492474.07216213,4217796.75094645));
+#49=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#50);
+#50=IFCCIRCULARARCSEGMENT2D(#51,1.06514750361384,497.87228283,1387.18510508,.T.);
+#51=IFCCARTESIANPOINT((492919.03457191,4218254.04591037));
+#52=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#53);
+#53=IFCTRANSITIONCURVESEGMENT2D(#54,1.42405564430839,461.366,1387.18510508,0.,.T.,.F.,.CLOTHOIDCURVE.);
+#54=IFCCARTESIANPOINT((493077.718,4218723.137));
+#55=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#56);
+#56=IFCLINESEGMENT2D(#57,1.59035139038631,100.);
+#57=IFCCARTESIANPOINT((493094.24,4219183.64));
+#58=IFCALIGNMENT2DVERTICAL(());
```

Comment:
Since providing `IfcAlignment2DVertical` in an `IfcAlignmentCurve` is optional, it would be better to not have an empty entity here, but rather `$` there.

Reply:
Removed.